docs/content/dagster-cloud/managing-deployments/managing-multiple-projects-and-teams.mdx

```diff
@@ -0,0 +1,177 @@
+---
+title: Managing multiple projects & teams in Dagster Cloud
+description: "How to set up and manage multiple projects with Dagster Cloud Hybrid deployments."
+---
+
+# Managing multiple projects & teams in Dagster Cloud
+
+<Note>This guide is applicable to Dagster Cloud.</Note>
+
+Whether to manage complexity or to divide your work into areas of responsibility, you may want to isolate your code bases into multiple projects. See [Strategies for code separation](#strategies-for-code-separation) for more information.
+
+If you require isolation between your execution context, see [Strategies for execution context isolation](#strategies-for-execution-context-isolation).
+
+## Strategies for code separation
+
+<Note>
+  Disambiguation: In this section, repository refers to a depot under a version
+  control system such as Git or Mercurial.
+</Note>
+
+### Multiple directories in a single repository
+
+You can use a single repository to manage multiple projects by placing each project in a separate directory. Depending on your VCS, you may be able to set [code owners](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners) to restrict who can modify each projects.
```

Comment:
I had to look up VCS - consider dropping "Depending on your VCS,"

s/"each projects"/"each project"